src/osd/PeeringState.cc

```diff
@@ -1607,22 +1607,21 @@ bool PeeringState::recoverable_and_ge_min_size(const vector<int> &want) const
           pool.info.is_erasure() ? shard_id_t(i) : shard_id_t::NO_SHARD));
     }
   }
-  // We go incomplete if below min_size for ec_pools since backfill
```

Comment:
Should you update recoverable_and_ge_min_size()?

Reply:
thank you, @dzafman, how about `PeeringState::recoverable` ?

Reply:
sure